calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java

```diff
@@ -4041,6 +4041,11 @@ public String buildIcalExportPanelContext(VelocityPortlet portlet, Context conte
 		boolean exportEnabled = CalendarService.getExportEnabled(calId);
 		context.put("enable_export", String.valueOf(exportEnabled) );
 
+		String icalInfoArr[] = {String.valueOf(ServerConfigurationService.getInt("calendar.export.previous.months",6)),
+			String.valueOf(ServerConfigurationService.getInt("calendar.export.next.months",6))};
```

Comment:
I think you have the arguments the wrong way round for the parameter substitution. You're bundle in plus/minus but your array is previous/next.